tenseal/cpp/tensors/bfvtensor.h

```diff
@@ -0,0 +1,193 @@
+#ifndef TENSEAL_TENSOR_BFVTENSOR_H
+#define TENSEAL_TENSOR_BFVTENSOR_H
+
+#include "tenseal/cpp/tensors/encrypted_tensor.h"
+#include "tenseal/cpp/tensors/plain_tensor.h"
+#include "tenseal/proto/tensors.pb.h"
+
+namespace tenseal {
+
+using namespace seal;
+using namespace std;
+
+class BFVTensor : public EncryptedTensor<int64_t, shared_ptr<BFVTensor>>,
+                  public enable_shared_from_this<BFVTensor> {
+   public:
+    using EncryptedTensor<int64_t, shared_ptr<BFVTensor>>::decrypt;
+    /**
+     * Create a new BFVTensor from a tensor of any shape.
+     * @param[in] input vector.
+     * @param[in] input vector.
+     * @param[in] input vector.
+     */
```

Comment:
Sorry, but the comments still don't make much sense.

Why the
```  * @param[in] input vector.
     * @param[in] input vector.
     * @param[in] input vector.```
     ?

Reply:
Ok @bcebere I'll change it accordingly, I just left the same comments which were already present in [CKKSTensor.h](https://github.com/OpenMined/TenSEAL/blob/1b371b678c804c47a993caee9088508b6a2bd1f5/tenseal/cpp/tensors/ckkstensor.h#L18) assuming they were correct.

Should I then change also the ones for CKKSTensor?

Reply:
Yes, please. Thank you so much for improving this!